packages/admin-ui-docs/docs/experimental/filter.mdx

```diff
@@ -5,6 +5,10 @@ path: /filter/
 
 # Filter
 
+Filters are supposed to be used in pages or sections that display some kind of data that can be filtered. The `filter` component is
+ a picker that displays filter options and allows for only one of them to be selected, when one of the options is selected it 
+ should produce a change on the data or content being displayed. It should be filtered accordingly.
```

Comment:
"It should be filtered accordingly." What is the subject of this?

Reply:
the data or content being displayed